src/Symfony/Bundle/DebugBundle/Resources/config/services.xml

```diff
@@ -38,7 +38,7 @@
             <argument>0</argument> <!-- flags -->
             <call method="setDisplayOptions">
                 <argument type="collection">
-                    <argument key="fileLinkFormat" type="service" id="debug.file_link_formatter"></argument>
+                    <argument key="fileLinkFormat" type="service" id="debug.file_link_formatter" on-invalid="ignore"></argument>
```

Comment:
If the service is invalid then the call still occurs with an empty collection as argument. I suggest to remove the call entirely and add it conditionally in DebugExtension instead